code/game/objects/items/grenades/flashbang.dm

```diff
@@ -16,6 +16,11 @@
 	new /obj/effect/dummy/lighting_obj (flashbang_turf, LIGHT_COLOR_WHITE, (flashbang_range + 2), 4, 2)
 	for(var/mob/living/M in get_hearers_in_view(flashbang_range, flashbang_turf))
 		bang(get_turf(M), M)
+	
+	for(var/obj/structure/blob/B in get_hear(8,flashbang_turf))     		//Blob damage here -- Yogs
+		var/distance = get_dist(B, get_turf(src))
+		var/damage = round(100/(distance*distance)+1)

```

Comment:
![image](https://user-images.githubusercontent.com/20558591/49317688-bd9abe00-f4f5-11e8-9f3a-589070b50ae4.png)

dividing by 0 runtimes, so if the flashbang is on the same turf this'll break :)

Reply:
To be fair it’s impossible to get a blob and a flashbang on the same tile